WordPress/Info.plist

```diff
@@ -240,8 +240,6 @@
 	</array>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
-	<key>MinimumOSVersion</key>
-	<string>4.0</string>
```

Comment:
🤔  not sure why that got removed (and on which PR)… probably during the migration to Xcode 12? 🤷